### PR TITLE
GH-859 Add `%eternalcore_afk_formatted%` placeholder.

### PR DESCRIPTION
**Pull request by vLuckyyy**
## Summary by CodeRabbit

- **New Features**
	- Introduced new AFK status placeholders for both English and Polish translations.
	- Enhanced messaging capabilities related to player status, including new messages for inactivity and player deaths.
	- Added support for retrieving AFK status messages through new translation methods.

- **Bug Fixes**
	- Improved clarity and functionality in placeholder registration for AFK status.

- **Documentation**
	- Updated translation interfaces to include new methods for AFK status messages.

### CODE REVIEW

**Review by CitralFlo — COMMENTED** [review]
Code looks good. What's the use case for that?

**Comment by vLuckyyy** [comment]
> Code looks good. What's the use case for that?

@Kryspek asked to add, use this I want to for information whether a player is afk or not over the player's head in a custom plugin for overhead holograms.

File: eternalcore-core/src/main/java/com/eternalcode/core/translation/implementation/PLTranslation.java [hunk 1]
```diff
@@ -511,6 +511,9 @@ public static class PLAfkSection implements AfkSection {
 
         @Description({ " " })
         public String afkKickReason = "<red>Zostałeś wyrzucone z powodu braku aktywności!";
+
+        public String afkEnabledPlaceholder = "<red><b>AFK";
```
Comment (Rollczi) [inline eternalcore-core/src/main/java/com/eternalcode/core/translation/implementation/PLTranslation.java:515]: Describe what placeholders are used for

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AftPlaceholderSetup.java [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
+    }
+
     @Subscribe(EternalInitializeEvent.class)
     void setUpPlaceholders(PlaceholderRegistry placeholderRegistry, AfkService afkService) {
-        placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of("afk", player -> String.valueOf(afkService.isAfk(player.getUniqueId()))));
+        placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of(
+            "afk",
+            player -> String.valueOf(afkService.isAfk(player.getUniqueId()))));
+        placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of(
+            "afk_text",
```
Comment (Rollczi) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AftPlaceholderSetup.java:32]: suffix `_text` doesn't describe what the placeholder does

Comment (vLuckyyy) [reply eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AftPlaceholderSetup.java:32]: give me a better idea, please

Comment (CitralFlo) [reply eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AftPlaceholderSetup.java:32]: maybe `afk_status` ?

Comment (vLuckyyy) [reply eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AftPlaceholderSetup.java:32]: then the placeholder name of another placeholder %eternalcore_afk% doesn't make sense, one is supposed to return formatted status and the other returns a boolean

**Review by P1otrulla — APPROVED** [review]

**Review by CitralFlo — APPROVED** [review]
LGTM

**Review by Rollczi — APPROVED** [review]

**Pull request merged**